src/user-defined-types/let-control-flow.md

```diff
@@ -0,0 +1,122 @@
+---
+minutes: 3
+existing course material:
+- control-flow/novel.md
+- control-flow/if-let-expressions.md
+- control-flow/while-let-expressions.md
+---
+
+# Let Control Flow
+
+Rust has a few control flow constructs which differ from other languages. They
+are used for pattern matching:
+
+- `if let` expressions
+- `while let` expressions
+- `match` expressions
+# `if let` expressions
```

Comment:
```suggestion
# `let-else` expressions
```

We've already introduced `match`, and presumably the commented-out case here should be about let-else which is something of a minor complication to `let` that makes sense after seeing if-let nesting's rightward drift.